### ISSUE TITLE
Plugin loads, but doesn't do anything

### ISSUE THREAD

**Issue by cynical-cyborg** [occ-ai/obs-backgroundremoval#576]
**Describe the bug**

Plugin loads in effects, but doesn't do anything

**To Reproduce**

1. Select 'Background Removal' filter under 'effects'

**Expected behavior**

Background removes and plugin functions.

**Screenshots**

![removal](https://github.com/occ-ai/obs-backgroundremoval/assets/168025119/e8373175-e9f7-4299-855d-5ed51b241098)

**Log and Crash Report**

[2024-04-24 12-56-59.txt](https://github.com/occ-ai/obs-backgroundremoval/files/15099409/2024-04-24.12-56-59.txt)

**Desktop (please complete the following information):**

 - OS: Windows 10 Pro
 - Browser: FireFox
 - Plugin Version: 1.1.13
 - OBS Version: OBS Studio 30.1.2

**Additional context**

I've tried uninstalling and reinstalling the plugin multiple times. The plugin was installed using the .exe installer.

**Comment by umireon** [comment]
Can you try our plugin with Windows 11?

**Comment by cynical-cyborg** [comment]
> Can you try our plugin with Windows 11?

Unfortunately, all of my computers are hardware-restricted from upgrading to Windows 11 at the moment.

**Comment by umireon** [comment]
I don't think we can support your hardware because it is too old but please try the older version of our plugin.
https://github.com/occ-ai/obs-backgroundremoval/releases/tag/1.1.8

**Comment by royshil** [comment]
the CPU is from 12 years ago https://www.intel.com/content/www/us/en/products/sku/71670/intel-core-i73632qm-processor-6m-cache-up-to-3-20-ghz-bga/specifications.html
and unfortunately not supported by our neural network inference engine

**Comment by cynical-cyborg** [comment]
Ah, gotcha. Thanks for the response and help!
